site/dlx.xml

```diff
@@ -230,9 +230,10 @@ args.put("x-dead-letter-routing-key", "some-routing-key");</pre>
             expiring again in any queues it is routed to.</li>
         </ul>
 
-        In case <code>x-death</code> already contains an entry with
+        New entries are prepended to the beginning of the <code>x-death</code>
+        array. In case <code>x-death</code> already contains an entry with
         the same queue and dead lettering reason, its count field will be
-        incremented.
+        incremented and it will be moved to the beginning of the array.
```

Comment:
@EdeMeijer I'd need to check [the code](https://github.com/rabbitmq/rabbitmq-server/blob/stable/src/rabbit_dead_letter.erl) since I don't remember off the top of my head. Well, the existing entries are updated so it's a matter of wording.

The focus of https://github.com/rabbitmq/rabbitmq-server/issues/78 wasn't so much to enforce a particular ordering.